packages/scandipwa/src/component/CheckoutOrderSummaryPriceLine/CheckoutOrderSummaryPriceLine.component.js

```diff
@@ -52,10 +56,30 @@ export class CheckoutOrderSummaryPriceLine extends PureComponent {
         );
     }
 
+    renderTitle() {
+        const { title, mods: { couponCode } } = this.props;
+
+        if (couponCode) {
+            return (
+            <p block="CheckoutOrderSummary" elem="Text">
+                { title }
+                <b>
```

Comment:
You can avoid code duplicates by moving coupon render logic to a small separate method:
``` 
renderCoupon(){
const  { couponCode } = this.props;

 if (!couponCode){
    return null;
 }
 
return  <b>
            { ` ${ couponCode.toUpperCase() }:` }
       </b>
  }
}

Keep in my that the ScandiPWA methods are intended to be overridden => the smaller they are, the better.